agent/cloudinit/file_writer_test.go

```diff
@@ -97,4 +97,40 @@ var _ = Describe("FileWriter", func() {
 		Expect(string(buffer)).To(Equal(fileOriginContent + file.Content))
 
 	})
+
+	It("should return error with invalid owner format", func() {
+		filePermission := 0777
```

Comment:
Why the file permission is `777` and not `755`. The agent scripts will only be executed by the owner or anyone?